src/pages/Defi/components/OverviewHeader.tsx

```diff
@@ -50,7 +52,7 @@ export const OverviewHeader = ({
               <Text translation='defi.netWorth' />
             </StatLabel>
             <StatNumber fontSize={48}>
-              <Amount.Fiat value={walletBalance} />
+              <Amount.Fiat value={bnOrZero(netWorth).plus(bnOrZero(lpBalance)).toString()} />
```

Comment:
- `netWorth` should always be defined, so that should work?
- `lpbalance` has a default value of `'0'`
```suggestion
              <Amount.Fiat value={bn(netWorth).plus(bn(lpBalance)).toString()} />
```